config/templates/stack-template.json

```diff
@@ -17,19 +17,19 @@
           "{{$subnet.AvailabilityZone}}"
           {{end}}
         ],
-        "DesiredCapacity": "{{.WorkerCount}}",
+        "DesiredCapacity": "{{.WorkerASG.MinSize}}",
```

Comment:
Yes. However if that's your intention it should be achieved in an another way.

[`DesiredCapacity` is not a required parameter](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html#cfn-as-group-desiredcapacity).
If you want to leave your initial ASG size as same as MinSize, you could just omit `"DesiredCapacity": "{{.WorkerASG.MinSize}}",` so stack-template would look like:

```json
{{if gt .WorkerCount 0}}"DesiredCapacity": "{{.WorkerCount}}",{{end}}
```

With this, If workerCount is omitted from cluster.yaml, `DesiredCapacity` is automatically set to MinSize by ASG itself and we're even unable to update DesiredCapacity via cfn stack updates. Thus, now, the user is responsible to update DesiredCapacity outside of kube-aws or CloudFormation.